docs/types.rst

```diff
@@ -568,7 +573,7 @@ Forced data location:
  - state variables: storage
 
 Default data location:
```

Comment:
This whole list can be removed, I think.

Reply:
Data location is either forced (state variables) and cannot be specified or it has to be specified and there is no default.